libraries/rating-bar/src/main/java/com/trendyol/uicomponents/ratingbar/BindingAdapters.kt

```diff
@@ -0,0 +1,16 @@
+package com.trendyol.uicomponents.ratingbar
+
+import android.widget.ImageView
+import androidx.annotation.ColorInt
+import androidx.core.graphics.drawable.DrawableCompat
+import androidx.databinding.BindingAdapter
+
+object BindingAdapters {
+
+    @BindingAdapter("tintCompat")
```

Comment:
library module should be Databinding free

Reply:
We will discuss this later.